LocalWiki.php

```diff
@@ -27,6 +27,38 @@ function onBeforePageDisplay( OutputPage $out ) {
 			$out->addMeta( 'og:image:width', '1200' );
 		}
 
+		break;
+	case 'gratisdatawiki':
+		$wgHooks['OutputPageParserOutput'][] = 'onOutputPageParserOutput';
+		
+		function onOutputPageParserOutput( OutputPage $out, ParserOutput $parserOutput ) {
+			$placeholders = $parserOutput->getExtensionData( 'wikibase-view-chunks' );
+			if ( $placeholders !== null ) {
+				$out->setProperty( 'wikibase-view-chunks', $placeholders );
+			}
+			$termsListItems = $parserOutput->getExtensionData( 'wikibase-terms-list-items' );
+			if ( $termsListItems !== null ) {
+				$out->setProperty( 'wikibase-terms-list-items', $termsListItems );
+			}
+			$meta = $parserOutput->getExtensionData( 'wikibase-meta-tags' );
+			$out->setProperty( 'wikibase-meta-tags', $meta );
+			$alternateLinks = $parserOutput->getExtensionData( 'wikibase-alternate-links' );
+			if ( $alternateLinks !== null ) {
+				foreach ( $alternateLinks as $link ) {
+					$out->addLink( $link );
+				}
+			}
+			
+			return true;
+		}
+		$wgHooks['BeforePageDisplay'][] = 'onBeforePageDisplay';
```

Comment:
Move to the top, so `$wgHooks` is kept together for nicer format.

Reply:
Done